### PR TITLE
use mysten histogram in consensus adapter

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

To avoid another metrics disruption, we add new metrics and will delete old ones later.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by mwtian — COMMENTED** [review]
I think the new metric format will require a new query anyway, so not sure adding a new metric with a suffix is meaningful?

**Comment by arun-koshy** [comment]
Might be better to make the breaking metrics changes now and then silence the alerts and swap the queries out on our end instead of having to make two code changes.

File: crates/sui-core/src/consensus_adapter.rs [hunk 1]
```diff
@@ -122,12 +133,23 @@ impl ConsensusAdapterMetrics {
                 registry,
             )
                 .unwrap(),
+            sequencing_certificate_latency_adapter: mysten_metrics::histogram::HistogramVec::new_in_registry(
+                "sequencing_certificate_latency",
```
[the remark this answers is not in the document]
Comment (longbowlu) [reply crates/sui-core/src/consensus_adapter.rs:137]: good catch. this is why the tests are failing.

**Comment by longbowlu** [comment]
@arun-koshy  I'm open to that. @mwtian do you have any preferences?

**Comment by longbowlu** [comment]
> @arun-koshy I'm open to that. @mwtian do you have any preferences?

I missed your comment earlier. Will just update the old metrics then

**Review by mwtian — APPROVED** [review]

**Review by akichidis — APPROVED** [review]

**Pull request merged**